.github/workflows/helm.yml

```diff
@@ -61,8 +63,29 @@ jobs:
         uses: helm/kind-action@v1.2.0
         if: steps.list-changed.outputs.changed == 'true'
 
+      - name: Install TimescaleDB in the cluster
+        if: steps.list-changed.outputs.changed == 'true'
+        run: |
+          helm repo add timescale 'https://charts.timescale.com'
+          helm repo update
+          helm install \
+            --set image.tag="pg14.3-ts2.7.0-p0" \
```

Comment:
Could you add a note that we need to remove this override after new timescaledb-single helm chart is released? Otherwise we won't be able to catch issues like recent ones.